models/Language.yaml

```diff
@@ -13,7 +13,7 @@ versions: String
 grammar_file: String
 aliases: Alias[]
 extensions: FileExtension[]
-line_continuation: String
+line_continuation: String[]
```

Comment:
Im pretty sure this doesnt work in statik, so take https://codein.withgoogle.com/dashboard/tasks/6456313849577472/ or https://codein.withgoogle.com/dashboard/tasks/4836154709901312/ to re-do this bit only, and ensure that all of the `line_continuation` in existing languages work with your schema enhancement.